calico-enterprise_versioned_docs/version-3.18-2/variables.js

```diff
@@ -1,12 +1,12 @@
 const releases = require('./releases.json');
 
 const variables = {
-  releaseTitle: 'v3.18.0-2.0',
+  releaseTitle: 'v3.18.1',
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.18',
   baseUrl: '/calico-enterprise/3.18',
```

Comment:
```suggestion
  baseUrl: '/calico-enterprise/latest',
```
ALSO: Change the equivalent 3.17 variables.js from latest -> 3.17

Reply:
Fixed.